manifests/resource/upstream/member.pp

```diff
@@ -85,7 +85,7 @@
     target  => "${conf_dir}/${upstream}-upstream.conf",
     order   => 40,
     content => epp('nginx/upstream/upstream_member.epp', {
-      _server        => $_server,
+      real_server    => $real_server,
```

Comment:
On the left hand side it can just be `server`. Right hand side can still be `$_server`